hms-app/src/components/input-forms/HackathonForm.tsx

```diff
@@ -27,33 +28,46 @@ function HackathonForm(props: IProps) {
 
   function createThisHackathon(event: React.MouseEvent<HTMLButtonElement>) {
     event.preventDefault()
-    showNotification({
-      id: 'hackathon-load',
-      loading: true,
-      title: 'Hackathon is uploading',
-      message: undefined,
-      autoClose: false,
-      disallowClose: true,
-    })
-    createHackathon(
-      instance,
-      hackathonTitle,
-      DescriptionValue,
-      startDateValue!,
-      endDateValue!
-    ).then((response) =>
-      setTimeout(() => {
-        console.log(response)
-        updateNotification({
+    if (startDateValue !== null && endDateValue !== null) {
+      if (endDateValue <= startDateValue) {
+        showNotification({
+          id: 'hackathon-end-before-start',
+          title: 'End date must be after start date',
```

Comment:
instead of this notification we should just disable the button when startdate > enddate

Reply:
That would be not user-friendly. There should be feedback.